packages/sewing-kit-koa/src/manifests.ts

```diff
@@ -0,0 +1,150 @@
+import {join} from 'path';
+
+import {readFile, readJson, pathExists} from 'fs-extra';
+import {matchesUA} from 'browserslist-useragent';
+import appRoot from 'app-root-path';
+import {ungzip} from 'node-gzip';
+
+import {ConsolidatedManifest, Manifest} from './types';
+
+const DEFAULT_MANIFEST_PATH = 'build/client/assets.json';
+const MULTI_ASYNC_LANGUAGE_MANIFESTS = 'noLocales';
+
+export interface ResolveOptions {
+  locale?: string | undefined;
+}
+
+export class Manifests {
+  manifestPath: string;
```

Comment:
same with some of this stuff, since this new class is well named and specific I think we can make a lot of stuff less wordy in here by deleting the word `manifest`

Reply:
Chopped some stuff out 👍 Probably still a bit wordy in places, but I tried to let go 😄